From bc86554bbda0582fe64fb46742e1789a348223ba Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago@kde.org>
Date: Tue, 1 Dec 2009 19:28:22 +0100
Subject: [PATCH 10/12] Update README file.

There's no more Qt Software and the bug reporting system is now JIRA
at http://bugreports.qt.nokia.com
---
 README.kde-qt |   17 +++++++++--------
 1 files changed, 9 insertions(+), 8 deletions(-)

diff --git a/README.kde-qt b/README.kde-qt
index db3feb6..6a10def 100644
--- a/README.kde-qt
+++ b/README.kde-qt
@@ -132,20 +132,21 @@ may be lost in a future update if they have not been added to the
 official Qt release.
 
 The exception to the above rule is that if the fix has been accepted
-by Qt Software (and so will appear in the very next release of Qt),
-then it should be simply cherry-picked from the Qt development
+by the Qt developers (and so will appear in the very next release of
+Qt), then it should be simply cherry-picked from the Qt development
 branch. Note that you shouldn't do this for changes that have been
-accepted into a release which is not the very next.
-In this case, you should use the following command:
+accepted into a release which is not the very next.  In this case, you
+should use the following command:
 
    git cherry-pick -x SHA1_OF_THE_FIX
 where SHA1_OF_THE_FIX is the SHA-1 of the commit that you want to
 introduce. Then push the change to the server.
 
-Before creating a patch, it is recommended to contact Qt Software
-support via qt-bugs@trolltech.com and explain the situation. There may
-be a solution for the problem already or a new direction that should
-be accounted for.
+In all other cases, before creating a patch, it is recommended to
+contact the Qt developers via a new task in
+http://bugreports.qt.nokia.com and explain the situation. There may be
+a solution for the problem already or a new direction that should be
+accounted for.
 
 To create a patch, do the following:
   a) look at the listing of branches in
-- 
1.6.5.6

